src/framegrab/grabber.py

```diff
@@ -559,6 +559,29 @@ def __init__(self, config: dict):
         self.idx = idx
         GenericUSBFrameGrabber.indices_in_use.add(idx)
 
+    def _validate_image_capture(self, capture: cv2.VideoCapture) -> bool:
+        """Check if the camera is able to grab a frame and that frame is a color frame.
+        We are excluding grayscale cameras in order to avoid connecting to IR cameras on webcams such
+        as the Logitech Brio
+        """
+        ret, frame = capture.read()
```

Comment:
Will capture.read perform the same check that capture.isOpened will? What happens if we try to read from a capture that isn't open?

Reply:
`capture.read` performs the same operation that `capture.isOpened`, but it also gives us a frame that we can analyze to see if it is grayscale. 

If the capture is not opened, capture.read will return False, None